app/scenes/Search/components/UserFilter.tsx

```diff
@@ -22,13 +23,17 @@ function UserFilter(props: Props) {
 
   const options = React.useMemo(() => {
     const userOptions = users.all.map((user) => ({
+      user,
       key: user.id,
       label: user.name,
+      icon: <Avatar model={user} />,
```

Comment:
@tommoor right now it's showing default ring on author. should we change it to something else?